### ISSUE TITLE
Perhaps FileHandle should be !Send and !Sync on every platform

### ISSUE THREAD

**Issue by Andrepuel** [PolyMeilex/rfd#63]
On linux, the type FileHandle is Send and Sync. On wasm32 FileHandle is !Send and !Sync.

It makes a little bit harder to do multiplatform code because a code that worked on Linux may not compiled on Wasm32.

**Comment by PolyMeilex** [comment]
Indeed, I think we can make it !Send !Sync, and if someone does not care about wasm and needs Send/Sync, they can just unwrap the file handle and use Path directly.
